prac_05/hex_colours.py

```diff
@@ -0,0 +1,16 @@
+"""
+CP1404/CP5632 Practical
+"""
+
+COLOUR_TO_CODE = {"absolutezero": "#0048ba", "amaranth": "#e52b50", "aquamarine1": "#7fffd4", "aquamarine2": "#76eec6",
+                  "blue1": "#0000ff", "blue2": "#0000ee", "celadon": "#ace1af",
+                  "ceruleanfrost": "#6d9bc3", "caribbeangreen": "#00cc99", "cadetblue1": "#98f5ff"}
+print(COLOUR_TO_CODE)
+colour_name = input("Enter colour: ").lower()
+while colour_name != "":
```

Comment:
Small suggestion. If you place .lower() in the while loop, you won't have to include it every time...